onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -523,6 +604,11 @@ static bool IsBinaryOpQuantizedInputsSupported(const Node& node) {
   return true;
 }
 
+Status GetQuantizedInputScaleAndZeroPoint(const ModelBuilder& model_builder,
+                                          const Node& node,
+                                          const std::string& input_name,
+                                          float& scale,
+                                          int32_t& zero_point) ORT_MUST_USE_RESULT;
 Status GetQuantizedInputScaleAndZeroPoint(const ModelBuilder& model_builder,
                                           const Node& node,
```

Comment:
static?

Reply:
Will correct in my other PRs

---
In reply to: [465459906](https://github.com/microsoft/onnxruntime/pull/4701#discussion_r465459906) [](ancestors = 465459906)